ChangeLog.d/rename_psa_crypto_driver_wrappers.txt

```diff
@@ -0,0 +1,3 @@
+Changes
+   * The auto-generated file library/psa_crypto_driver_wrappers.c has
+     been renamed to psa_crypto_driver_wrappers_no_static.c.
```

Comment:
It might also be useful to add a line like "(This may affect people who are integrating their own PSA drivers with Mbed TLS)." as otherwise people not affected and unfamiliar with drivers may be confused as to why this is in the changelog at all, and whether it's something they need to care about.

Reply:
I will be be adding Dave's suggestion or a derivative of it. So the entry witll contain:

- The change
- The context

which makes it more user friednly.